src/components/course/edit/sections/TimeSection.vue

```diff
@@ -1,49 +1,28 @@
 <template>
-    <section class="border-t-2 py-8 border-gray-400">
-        <div class="lg:flex">
-            <div class="w-full lg:w-1/3 lg:block mr-12 flex flex-col mb-4">
-                <label class="block text-gray-700 text-md font-medium mb-2">Time</label>
-                <label class="block text-gray-600">
-                    Information about Start and End Dates of the Course as well as Slot Times. This section is disabled for now as there is
-                    no Vue3 datepicker plugin yet.
-                </label>
+    <base-section title="Time" subtitle="Manage dates as well as time slots.">
+        <div class="lg:flex lg:space-x-12 lg:space-y-0 space-y-4 w-full">
+            <div class="lg:w-1/2 w-full">
+                <label class="input-label-tmp">Start Date</label>
+                <input id="startDate" class="input-text-tmp w-full" readonly type="text" :value="start" />
```

Comment:
Read-only fields should always have the ability to be focused so it's clear when they are selected in case someone wants to copy its content especially with keyboard only. However, if we use disabled instead of read-only the focus style will vanish and this might be of good use here since these are just stubs anyway.